Shared/View/VoteView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  VoteView.swift
+//  Fun Gen
+//
+//  Created by Angkana on 5/2/22.
+//
+
+import SwiftUI
+
+struct VoteView: View {
+    // Hardcoded values to test
+    @State private var tempActivty = Activity(
+        id: "123",
+        title: "What movie should we watch tonight?",
+        category: Category.movie,
+        author: "mightyDonut",
+        voteCount: 3
+    )
+    @State private var optionArray = ["Shrek", "Wall-E", "ET"]
+    @State private var isSelected = ""
+    @State private var newOption = ""
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text("Activity").font(.title).padding(.bottom)
+            Text(tempActivty.title).font(.title3)
+            Text("Category: \(tempActivty.category.rawValue.capitalized)").foregroundColor(.secondary)
+            Text("Options").font(.title).padding(.top)
+            // Option selection
+            ScrollView {
+                VStack(alignment: .leading) {
+                    // TODO: Correctly update fields from Model files when user inputs data
+                    // TODO: Update voteCount += 1 when an option is selected
+                    ForEach(optionArray, id: \.self) { item in
+                        Button(action: { self.isSelected = item }) {
```

Comment:
To fix this SwiftLint warning, you can use:
```suggestion
                        Button {
                            self.isSelected = item
                        } label: {
```